src/components/Dashboard/Flavors.js

```diff
@@ -2,34 +2,104 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import React, { Component } from 'react';
 import { bindActionCreators } from 'redux';
-import { Table } from 'react-bootstrap';
+import { Col, Container, Row, Table } from 'react-bootstrap';
 import { DashboardLayout as Layout } from 'components/Dashboard/';
 
 import { actions as flavorsActions } from 'reducers/flavors';
-import { getAllFlavors } from 'selectors/flavors';
+import { getAllFlavors, getFlavorsPager } from 'selectors/flavors';
 
 export class Flavors extends Component {
   static propTypes = {
     actions: PropTypes.object.isRequired,
     layoutOptions: PropTypes.object.isRequired,
-    flavors: PropTypes.array
+    collection: PropTypes.array,
+    pager: PropTypes.object
   };
 
+  constructor(props) {
+    super(props);
+    this.state = { limit: 20, page: 1 };
+    this.handlePageChange = this.changePage.bind(this);
+    this.handleLimitChange = this.changeLimit.bind(this);
+    this.handleLimitUpdate = this.updateLimit.bind(this);
+  }
+
   componentDidMount() {
     const { actions } = this.props;
 
-    actions.requestFlavors();
+    actions.requestFlavors({ page: 1, limit: 20 });
+  }
+
+  pagerCounter() {
+    const { pages } = this.props.pager;
+
+    let i = 1;
+
+    const pager = [];
+
+    while (i <= pages) {
+      pager[i] = i;
+      i++;
+    }
+    return pager;
+  }
+
+  changePage(page) {
+    const { actions, pager } = this.props;
+
```

Comment:
Use destructuring here, like `const { target: { value } } = page;`

Reply:
Updated; same in changeLimit